inspirehep/modules/records/wrappers.py

```diff
@@ -114,6 +114,21 @@ def publication_information(self):
 
         return pub_info_list
 
+    @property
+    def external_system_identifiers(self):
+        """External system identification information.
+
+        Returns a list that contains information on first of each kind of 
+        external_system_idenitfiers
+        """
+        ext_sys_id_list = []
+        seen = set()
+        for ext_sys_id in self.get('external_system_identifiers', []):
+            if ext_sys_id['schema'] not in seen:
+                seen.add(ext_sys_id['schema'])
+                ext_sys_id_list.append(ext_sys_id)
+        return ext_sys_id_list
```

Comment:
Why not https://github.com/inspirehep/inspire-utils/blob/a3d30224979f0a6ca78b48edb6cdfdb29c18ea0b/inspire_utils/dedupers.py#L43-L70 ?

Reply:
Scratch that, you're doing something different here.